OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -41,14 +41,16 @@ public void Killed(Actor self, AttackInfo e)
 
 			var valued = self.Info.TraitInfoOrDefault<ValuedInfo>();
 
-			// Default experience is 100 times our value
 			var exp = info.Experience >= 0
 				? info.Experience
-				: valued != null ? valued.Cost * 100 : 0;
```

Comment:
On a second thought this probably wants to use something like `DivideBy(upgrades.Key, 100)`

```csharp
internal static void DivideBy(ref string input, int factor)
{
   var before = Exts.ParseIntegerInvariant(input);
   input = (before / factor).ToString()
}
```